docs/operator.md

```diff
@@ -131,6 +132,41 @@ A strategy the Operator uses for [upgrades](update.md#more-on-upgrade-strategies
 | ----------- | ---------- |
 | :material-code-string: string     | `SmartUpdate`              | 
 
+## <a name="operator-unsafeflags-section"></a>Unsafe flags section
+
+The `unsafeFlags` section in the [deploy/cr.yaml  :octicons-link-external-16:](https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml) file contains various configuration options to prevent users from configuring a cluster with unsafe parameters.
+
+### `unsafeFlags.tls`
+
+Prevents users from configuring a cluster without TLS/SSL certificates (if `false`, the Operator will automatically change unsafe parameters to safe defaults).
```

Comment:
> the Operator will automatically change unsafe parameters to safe defaults
this is not true, operator won't change anything starting from this release. it'll just print error in logs and set cluster status to error